mscore/stafftextproperties.cpp

```diff
@@ -389,7 +389,7 @@ void StaffTextProperties::channelItemChanged(QTreeWidgetItem* item, QTreeWidgetI
       for (const NamedEventList& e : part->instrument(tick)->midiActions()) {
             QTreeWidgetItem* ti = new QTreeWidgetItem(actionList);
             if (e.name.isEmpty() || e.name == Channel::DEFAULT_NAME) {
-                  ti->setText(0, tr(Channel::DEFAULT_NAME));
```

Comment:
this never worked apparently, that string ('normal', prior to this PR) never made it to Transifex